metricbeat/modules.d/windows.yml.disabled

```diff
@@ -1,2 +1,8 @@
 - module: windows
-  period: 60s
+  metricsets: ["perfmon"]
+  period: 10s
+  perfmon.counters:
```

Comment:
While it would be a deviance from the defaults, I think that including an example would be very helpful to users. We have some in the [docs](https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-metricset-windows-perfmon.html#_configuration_14).